[2025-10-17T20:53:35.724+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_run_pipeline.dbt_init manual__2025-10-17T20:53:32.974715+00:00 [queued]>
[2025-10-17T20:53:35.733+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_run_pipeline.dbt_init manual__2025-10-17T20:53:32.974715+00:00 [queued]>
[2025-10-17T20:53:35.734+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-10-17T20:53:35.748+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): dbt_init> on 2025-10-17 20:53:32.974715+00:00
[2025-10-17T20:53:35.752+0000] {standard_task_runner.py:57} INFO - Started process 178 to run task
[2025-10-17T20:53:35.756+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dbt_run_pipeline', 'dbt_init', 'manual__2025-10-17T20:53:32.974715+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpk0gb10e3']
[2025-10-17T20:53:35.759+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask dbt_init
[2025-10-17T20:53:35.778+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-17T20:53:35.823+0000] {task_command.py:415} INFO - Running <TaskInstance: dbt_run_pipeline.dbt_init manual__2025-10-17T20:53:32.974715+00:00 [running]> on host eb8a9840e92a
[2025-10-17T20:53:35.948+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_run_pipeline' AIRFLOW_CTX_TASK_ID='dbt_init' AIRFLOW_CTX_EXECUTION_DATE='2025-10-17T20:53:32.974715+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-17T20:53:32.974715+00:00'
[2025-10-17T20:53:35.951+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-17T20:53:35.952+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/dbt_project && dbt debug --profiles-dir . || true']
[2025-10-17T20:53:35.962+0000] {subprocess.py:86} INFO - Output:
[2025-10-17T20:53:35.965+0000] {subprocess.py:93} INFO - /bin/bash: line 1: cd: /opt/***/dbt_project: No such file or directory
[2025-10-17T20:53:35.966+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-17T20:53:36.004+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=dbt_run_pipeline, task_id=dbt_init, execution_date=20251017T205332, start_date=20251017T205335, end_date=20251017T205336
[2025-10-17T20:53:36.024+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-10-17T20:53:36.052+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
